pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,45 @@
+{ gtk3, python3, hunspell, isocodes, libcanberra_gtk3, xorg.libxkbfile, libxkbcommon, python35Packages.pycairo, python35Packages.dbus-python, python35Packages.pygobject3, python35Packages.systemd, libudev, python35Packages.distutils_extra, gnome3.dconf, pkgconfig, xorg.libXtst
```

Comment:
That's strange, I fixed that in the previous commit, I guess I undid the change by accident.